src/AutoRest.CSharp/Common/AutoRest/Plugins/Configuration.cs

```diff
@@ -34,6 +34,14 @@ public static class Options
             public const string SkipSerializationFormatXml = "skip-serialization-format-xml";
             public const string DisablePaginationTopRenaming = "disable-pagination-top-renaming";
             public const string SuppressAbstractBaseClasses = "suppress-abstract-base-class";
+            public const string RemoveUnusedTypes = "remove-unused-types";
+        }
+
+        public enum UnusedTypeRemovalLevel
+        {
+            RemoveAll = 0,
```

Comment:
Please make the name to be consistent with the configuration

Reply:
Renamed accordingly